packages/flame/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ Refer to the [CONTRIBUTING guide](https://github.com/lightspeed/flame/blob/maste
 
 ## [Unreleased]
 
+### Breaking
+
+- Use React.forwardRef on `<Button>` and `<Switch>` components ([#75](https://github.com/lightspeed/flame/pull/75))
```

Comment:
I know this deemed "breaking" by React, but do we absolutely need to mark it as breaking for Flame itself? If yes, our `next` branch will stay on hold for quite a while. Or are we comfortable on bumping to `2.0.0` soon with the current changes on `next`?

Reply:
Yes, this needs to be a breaking. We're trying to follow semver here. If it's deemed as a breaking change by the React team, it's a breaking here.

2.0.0 isn't coming yet. We'll be releasing an RC first (which we legit should be doing more often).

2.0.0 is scheduled to go out in a month or so. The rate at which we release major changes should be on a 3 month basis. This'll give us time to plan what needs to come out, how betas/RC will be released and to communicate the next batch of changes.

Most of the breaking things introduced in the 3 month cycle are going to be pretty simple to fix. Mostly deprecating things that were planned to be deprecated anyways and small adjustments here and there to finally remove all the legacy cruft.

Reply:
Yup, fully agree on releasing RCs for major releases. This is somewhat mentioned in the [release guide](https://github.com/lightspeed/flame/blob/next/RELEASE.md#pre-releases), we should probably be clearer on that end that major releases will see most of the time have RCs before stable.